templates/DatasetMonitoring/v2/researchGroups.html.twig

```diff
@@ -70,6 +70,9 @@
                     tooltipInstance.hide();
                 });
             }
+        },
+        onRowClick: function(e) {
+            window.open(`${Routing.generate('pelagos_app_ui_dataland_default')}/${ e.data.udi }`);
```

Comment:
Add target, I know by default it is blank, but we want to be explicit in case it changes.